tests/end_to_end/test_vulnerability_detector/data/playbooks/teardown_playbook.yaml

```diff
@@ -0,0 +1,15 @@
+- name: Cleanup environment
```

Comment:
Please, rename the playbook to `teardown.yaml`. Also, we need to clean the `ossec.conf`.

Reply:
Done in 042087b6a5e452e63b51eba402508835b21dc6e3